colossalai/zero/low_level/low_level_optim.py

```diff
@@ -418,6 +434,11 @@ def step(self, closure=None):
         # update the parameters
         self.optim.step()
 
+        # release the moe grad
+        if len(self.param_groups) > len(self._working_param_groups):
+            for param in self.param_groups[-1]['params']:
+                param.grad = None
+
         # release the grad
```

Comment:
Can you integrate moe grad handle into this optimizer? As we won't have any grad handler anymore.

Reply:
this grad handle is only for pytorch model test. will not be used in our model

Reply:
Then can we move this to `tests/`? Engine will be deprecated soon.

Reply:
ok